QUEST.tsv

```diff
@@ -2391,48 +2391,48 @@ QUEST_20150803_002398	There is a manager that manages the fortress of the great
-QUEST_20150918_002410	
-QUEST_20150918_002411	
-QUEST_20150918_002412	
-QUEST_20150918_002413	
-QUEST_20150918_002414	
-QUEST_20150918_002415	
-QUEST_20150918_002416	
-QUEST_20150918_002417	
-QUEST_20150918_002418	
-QUEST_20150918_002419	
-QUEST_20150918_002420	
-QUEST_20150918_002421	
-QUEST_20150918_002422	
-QUEST_20150918_002423	
-QUEST_20150918_002424	
-QUEST_20150918_002425	
-QUEST_20150918_002426	
-QUEST_20150918_002427	
-QUEST_20150918_002428 Underground Chapel Dungeon
-QUEST_20150918_002429	
-QUEST_20150918_002430	
-QUEST_20150918_002431	
-QUEST_20150918_002432	
-QUEST_20150918_002433	
-QUEST_20150918_002434	
-QUEST_20150918_002435	
-QUEST_20150918_002436	
-QUEST_20150918_002437	
-QUEST_20150918_002438	
-QUEST_20150918_002439	
-QUEST_20150918_002440	
-QUEST_20150918_002441	
-QUEST_20150918_002442	
-QUEST_20150918_002443	
-QUEST_20150918_002444	
-QUEST_20150918_002445	
-QUEST_20150918_002446	
+QUEST_20150908_002402	Obtained the woods
+QUEST_20150918_002403	There are no missions for Today.
+QUEST_20150918_002404	There are only missions that seem difficult to you.{nl} Please come again next time.
+QUEST_20150918_002405	Historic Site Ruins
+QUEST_20150918_002406	Historic Site Ruins(Recommended LV : 90). Only you and your party members can enter together.{nl}Are you sure you want to enter?
+QUEST_20150918_002407	Right. Goddess Laima is the strangest of them all.{nl}No one knows when she disappeared. To be honest , I don't know if she's really a Goddess.{nl}
+QUEST_20150918_002408	Even if they say the world is doomed, our soldiers will guard this palce until the end. {nl}However, we won't be responsible for your dangerous activities so be careful.
+QUEST_20150918_002409	Welcome. Uska is waiting you in the central square of the Klaipeda. {nl}Well, is there anything that you are unsure of?
+QUEST_20150918_002410	But this is also my ordeal and mission that Laima has seen.. {nl}I will tell you Laima's message.{nl}
+QUEST_20150918_002411	Welcome! What do you need. {nl}Not like previous items, I made my best effort to collect.
+QUEST_20150918_002412	I heard that the Revelator appeared and saved the Mage Tower. {nl}I can't imagine what would have happened to Fedimian if the Mage Tower fell.. It's such a relief.{nl}Here, feel free to look around anytime.
+QUEST_20150918_002413	Fedimain used to be crowded pilgrims in the past. {nl}I heard someone protected the Cathedral so I'm hoping it will be lively as it was during it's glorious days.{nl}So, what kind of items are you looking for?
+QUEST_20150918_002414	The red haired wizard found the revelator to save the Mage Tower. {nl}Even when there were other Masters. He did save the Mage Tower so that was great but..{nl}Anyway, seeing is free so look around.
+QUEST_20150918_002415	Amazing, huh? The revelator who protected the Cathedral.{nl}So we decided to not increase the price even though we're in tough times. Take your time.
```

Comment:
I'd rewrite this to:
"Amazing, huh? The Revelator protected the Cathedral. {nl} Because of that, we decided to raise our prices even though we're in tough times. Take your time."